native/packages/react-native-bpk-component-nudger/src/__snapshots__/BpkNudger-test.android.js.snap

```diff
@@ -123,8 +123,12 @@ exports[`Android BpkNudger should disable theming if the required attribute is o
               Array [
                 Object {
                   "lineHeight": 16,
+                  "marginLeft": 4,
+                },
+                Object {
+                  "marginLeft": 0,
```

Comment:
Again, it's a result of the terrible styling logic in this component. But basically we apply `marginLeft: 4` to the icon and then override it if it's an `iconOnly` button as we don't want a margin there. I've switched the logic around for that slightly, which is why it's changed here.

The `marginRight` thing is a good call as it may affect RTL and I didn't check that. I'll check and confirm / fix if necessary.

Reply:
Just checked it and it still works fine with RTL. The reason I remove it is because when RTL is set we do apply a `marginRight` to leading buttons and I want to ensure that's wiped out for icon only ones.

I promise I will make all of this less horrible very soon!